device-types/Cisco/C9500-12Q.yaml

```diff
@@ -0,0 +1,48 @@
+---
+manufacturer: Cisco
+model: Catalyst 9500-12Q
+slug: cisco-c9500-12q
+part_number: C9500-12q
+u_height: 1
+is_full_depth: true
```

Comment:
Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Reply:
done